packages/search-input/src/types.ts

```diff
@@ -0,0 +1,48 @@
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { BaseFontSize } from '@leafygreen-ui/tokens';
+
+export const State = {
+  None: 'none',
+  Loading: 'loading',
+} as const;
+
+export type State = typeof State[keyof typeof State];
+
+export const SizeVariant = {
+  XSmall: 'xsmall',
+  Small: 'small',
+  Default: 'default',
+  Large: 'large',
+} as const;
+
+export type SizeVariant = typeof SizeVariant[keyof typeof SizeVariant];
+
+export interface SearchInputProps
+  extends HTMLElementProps<'input', HTMLInputElement> {
```

Comment:
SearchInput should accept `aria-label` as well, and not require `aria-labelledby` in that case

Reply:
Actually, I just saw your PR for the TextInput component. What was the discussion there? It would be nice to get on the same page on it before moving forward.

Reply:
Yes definitely valid to accept an `aria-labelledby` prop that relates the label and input field.